ibm/service/power/resource_ibm_pi_instance.go

```diff
@@ -822,7 +846,7 @@ func isWaitForPIInstanceAvailable(ctx context.Context, client *st.IBMPIInstanceC
 
 	stateConf := &resource.StateChangeConf{
 		Pending:    []string{"PENDING", helpers.PIInstanceBuilding, helpers.PIInstanceHealthWarning},
-		Target:     []string{helpers.PIInstanceAvailable, helpers.PIInstanceHealthOk, "ERROR", ""},
+		Target:     []string{helpers.PIInstanceAvailable, helpers.PIInstanceHealthOk, "ERROR", "", "SHUTOFF"},
```

Comment:
As per your code, it is never returning SHUTOFF.

Reply:
It is returning `SHUTOFF` since that is one the target in the list. Given the refresFunc is querying for it.

Reply:
You are returning `helpers.PIInstanceAvailable`, not `SHUTOFF`.
```
		if (*pvm.Status == "SHUTOFF") && (pvm.Health.Status == instanceReadyStatus || pvm.Health.Status == helpers.PIInstanceHealthOk) {
			return pvm, helpers.PIInstanceAvailable, nil
		}
```

Reply:
Move it here `isWaitForPIInstanceShutoff`